easybuild/easyconfigs/c/cutadapt/cutadapt-1.16-goolf-1.7.20-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```

Comment:
should be `easybuilders`, not `hpcugent` (anymore), fixed in https://github.com/pescobar/easybuild-easyconfigs/pull/66